mantidproject/mantid
# Plots no longer rescale automatically when curve properties are changed.

**Description of work.**
Previously changing the axes scales of a plot and then using the figure options menu to change the curve properties would cause the axes to reset to their original scale. This is now fixed.

**To test:**
1. Plot a figure, double click on the axes to change their min and max values.
2. Click the cog icon to open figure options, go to the curves tab and change something.
3. Click apply and check that the axes scales haven't reset.

Fixes #26676 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
